Source/WebKit/Shared/DocumentEditingContext.mm

```diff
@@ -40,7 +40,31 @@ static inline NSRange toNSRange(DocumentEditingContext::Range range)
     return NSMakeRange(range.location, range.length);
 }
 
-UIWKDocumentContext *DocumentEditingContext::toPlatformContext(OptionSet<DocumentEditingContextRequest::Options> options)
+#if HAVE(UI_WK_DOCUMENT_CONTEXT)
+
+template <typename ContextType>
+void setOptionalEditingContextProperties(const DocumentEditingContext& context, ContextType *platformContext, OptionSet<DocumentEditingContextRequest::Options> options)
+{
+    for (auto& rect : context.textRects)
```

Comment:
Can this be a const reference?

Reply:
We don't normally use `const auto` in WebKit. I'd rather we keep `auto&`.